components/Layouts/Sidebar/data.tsx

```diff
@@ -1,22 +1,33 @@
 import { FaHome, FaNewspaper, FaUserFriends } from 'react-icons/fa';
-
+const iconStyleProjects = {
```

Comment:
Login/signup button icon is not same size as other icons. fix it as well. 

<img width="218" alt="Screenshot 2023-08-14 at 10 39 18 PM" src="https://github.com/rohitdasu/projectmate/assets/48400770/46d9a36f-8be6-40c1-8045-c9483c4698ea">